File: src/Neuralm.Services/Neuralm.Services.TrainingRoomService/Neuralm.Services.TrainingRoomService.Messages/EndTrainingSessionResponse.cs
using Neuralm.Services.Common.Messages.Abstractions;
using System;

namespace Neuralm.Services.TrainingRoomService.Messages
{
    /// <summary>
    /// Represents the <see cref="EndTrainingSessionResponse"/> class.
    /// </summary>
    public class EndTrainingSessionResponse : Response
    {
        /// <summary>
        /// Initializes an instance of the <see cref="EndTrainingSessionResponse"/> class.
        /// </summary>
        /// <param name="requestId">The request id.</param>
        /// <param name="message">The message.</param>
        /// <param name="success">The success flag.</param>
        public EndTrainingSessionResponse(Guid requestId, string message, bool success = false) : base(requestId, message, success)
        {

        }
    }
}

File: src/Neuralm.Presentation.Web/src/messages/responses/EndTrainingSessionResponse.ts
import Response from './Response';

/**
 * Represents the EndTrainingSessionResponse class.
 */
export default class EndTrainingSessionResponse extends Response {
  /**
   * Initializes a new instance of the EndTrainingSessionResponse class.
   */
  constructor(id: string, requestId: string, dateTime: Date, message: string, success: boolean) {
    super(id, requestId, dateTime, message, success);
  }
}
